app/package.json

```diff
@@ -37,6 +37,7 @@
         "remove_tag": "./scripts/remove_tag.sh"
     },
     "dependencies": {
+        "@onlook/onlook": "file:",
```

Comment:
There seems to be a lot of extra dependencies added to the PR. Can you remove these?

Reply:
ok

Reply:
Yep, let me add some comments